ENIGMAsystem/SHELL/Graphics_Systems/None/OPENGLStd.h

```diff
@@ -0,0 +1,429 @@
+/** Copyright (C) 2017 Faissal I. Bensefia
```

Comment:
Does the user have any way of observing that we're writing data into these vectors? Id est, is this necessary?

Reply:
I forgot to delete this file, nothing in here is being used for anything. I was mostly using it as a scratchpad. The 'real' version of the 3D stuff is in NONEmodel.cpp and NONEModelStruct.h, there is not *currently* a way of observing this data, but I've kept them so that once the d3d_model_save() function is actually implemented it can be used.